src/pages/api/users/[id].ts

```diff
@@ -0,0 +1,62 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+import { prisma } from '../../../../utils/prisma';
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  if (req.method === 'GET') {
+    const params = req.query;
```

Comment:
should be type string
```suggestion
    const string params = req.query;
```